pandas/core/groupby/ops.py

```diff
@@ -102,7 +102,7 @@ def groupings(self) -> List["grouper.Grouping"]:
         return self._groupings
 
     @property
-    def shape(self):
+    def shape(self) -> Tuple[int, ...]:
```

Comment:
@WillAyd Had to add this ellipsis here; any idea why mypy likes this but not just `Tuple[int]`?

Reply:
`Tuple[int]` means a tuple of size one containing an interger. `Tuple[int, ...]` means a tuple of an arbitrary size containing integers

I think should be `Tuple[int, int]` no? i.e. a tuple with two integers

Reply:
My guess is that `self.groupings` has arbitrary length so the return type needs to also